packages/core/contracts/optimism/FinderFlattened.sol

```diff
@@ -0,0 +1,156 @@
+/**
+ * @notice This contract was flattened so that it no longer imports directly or indirectly contracts that cannot be
```

Comment:
This is just a flattened Finder.sol that strips out external contract imports so that I can compile it successfully.